sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -229,4 +226,43 @@ export class EducationProgramControllerService {
       hasOfferings,
     };
   }
+
+  /**
+   * Allows a program to be deactivated.
+   * @param programId program to be updated.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param isActive value to update the isActive state for the program.
+   * @param options method options.
+   * - `institutionId` institution used for authorization.
+   * - `notes` notes associated with the change.
+   */
+  async deactivateProgram(
+    programId: number,
+    auditUserId: number,
+    options?: {
+      institutionId?: number;
+      notes?: string;
+    },
+  ): Promise<void> {
+    try {
+      await this.programService.updateEducationProgramIsActive(
+        programId,
+        auditUserId,
+        false,
+        options,
+      );
+    } catch (error: unknown) {
+      if (error instanceof CustomNamedError) {
+        if (error.name === EDUCATION_PROGRAM_NOT_FOUND) {
+          throw new NotFoundException(error.message);
+        }
+        if (error.name === EDUCATION_PROGRAM_INVALID_OPERATION) {
+          throw new UnprocessableEntityException(
+            new ApiProcessError(error.message, error.name),
```

Comment:
Is there an intention to display the error message in UI?

Reply:
Not really, I can remove the `ApiProcessError`.

Reply:
Removed.